rasa/core/channels/rest.py

```diff
@@ -58,6 +58,21 @@ def _extract_message(self, req: Request) -> Optional[Text]:
     def _extract_input_channel(self, req: Request) -> Text:
         return req.json.get("input_channel") or self.name()
 
+    def get_metadata(self, request: Request) -> Optional[Dict[Text, Any]]:
+        """Extracts additional information from the incoming request.
+
+         Implementing this function is not required. However, it can be used to extract
+         metadata from the request. The return value is passed on to the
+         ``UserMessage`` object and stored in the conversation tracker.
+
+        Args:
+            request: incoming request with the message of the user
+
+        Returns:
+            Metadata which was extracted from the request.
+        """
+        return request.json.get("metadata", None)
+
     def stream_response(
```

Comment:
Could you please add the missing docstring here so the code quality checks pass 🙏🏻

Reply:
I don't understand the error after reading up on flake8 and public method docstring requirements. I see lines 62-73 look like a valid docstring on a public method but clearly I'm wrong about that.

Reply:
Oh the linter is complaining about the `stream_response` method lacking a docstring. I know you haven't updated this method, however it just seems the linter will flag these past issues in the module you're working in.

Reply:
Thanks, fixed